src/utils/log.js

```diff
@@ -46,13 +46,16 @@ const banner = `
 | | | / __| | '_ \\ / _\` |   | |/ _ \\ '_ \` _ \\| '_ \\| |/ _\` | __/ _ \\
 | |_| \\__ \\ | | | | (_| |   | |  __/ | | | | | |_) | | (_| | ||  __/
  \\___/|___/_|_| |_|\\__, |   |_|\\___|_| |_| |_| .__/|_|\\__,_|\\__\\___|
-                   |___/                     |_|              v${pack.version}
-`;
+                   |___/                     |_|              v${pack.version}`;
 
-const goToStep = (step) => {
+const goToStep = (step, templateBanner = undefined) => {
```

Comment:
Should we make `templateBanner` global so it doesn't have to be passed in every step ?

Reply:
I think i'd rather keep it pure, its only restated a couple of times.